frontend/plan/components/schedule/Schedule.tsx

```diff
@@ -85,7 +85,7 @@ const ScheduleBox = styled.div`
     color: #4a4a4a;
     display: block;
     padding: 1.25rem;
-    height: calc(100vh - 12em);
+    min-height: calc(100vh - 12em);
 `;
```

Comment:
Seems like adding a course to the schedule expands the schedule box height, which makes the height of the boxes unequal. Why do we need `min-height` here?

<img width="1483" alt="Screen Shot 2022-09-06 at 8 32 53 PM" src="https://user-images.githubusercontent.com/44344658/188763217-be0059d2-5f0f-4791-8e82-8409ca47ac79.png">

Reply:
min height allows for expansion on mobile, where the viewport height may not be sufficient to hold the stats at the bottom. I'll create a new styled component for mobile or add a media query.